pkg/matrix/matrix.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package matrix
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"strconv"
```

Comment:
linting might fail here 🙃

Reply:
locally, I get a linting failure when I flip the order of imports - let's see what happens here when the tests run 🤔

Reply:
it [failed](https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_pipeline/4934/pull-tekton-pipeline-build-tests/1532764258557759488/) linting 🙃 

the imports settings in my ide was using gofmt instead of goimports, fixed and updated the imports, thanks @pritidesai!